From eaf8d6d1647a55fea2f76185544799b891aa16a5 Mon Sep 17 00:00:00 2001
From: Martchus <martchus@gmx.net>
Date: Sun, 26 Nov 2023 17:06:06 +0100
Subject: [PATCH] Mention PLASMA_ENABLE_QML_DEBUG in the Plasmoid testing docs

This variable might be useful and now seems to be supported by
`plasmawindowed`, see
https://invent.kde.org/plasma/plasma-workspace/-/merge_requests/3237.
---
 plasmoid/testing.md | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/plasmoid/testing.md b/plasmoid/testing.md
index f80aa66..361e31d 100644
--- a/plasmoid/testing.md
+++ b/plasmoid/testing.md
@@ -48,6 +48,9 @@ It works quite well within a Plasma session when both shells are placed on diffe
 edges.
 
 ## Enable QML debugging
+It is not clear whether the following instructions are still valid for Plasma 6.
+It seems that QML debugging can be enabled under Plasma 6 by setting the environment variable
+`PLASMA_ENABLE_QML_DEBUG` but this has not been tested yet.
 
 To enable QML debugging, it is required to rebuild `plasmoidviewer` with QML debugging
 enabled.
-- 
2.43.0

